compiler/rustc_passes/src/check_attr.rs

```diff
@@ -1101,17 +1101,6 @@ impl Visitor<'tcx> for CheckAttrVisitor<'tcx> {
         intravisit::walk_arm(self, arm);
     }
 
-    fn visit_macro_def(&mut self, macro_def: &'tcx hir::MacroDef<'tcx>) {
```

Comment:
This was just moved to lower in the file during rebasing. I can move it back if desired.